pocs/focuser/focuser.py

```diff
@@ -149,6 +149,8 @@ def autofocus(self,
                   take_dark=None,
                   merit_function=None,
                   merit_function_kwargs=None,
+                  mask_dilations=None,
+                  spline_smoothing=None,
```

Comment:
You mention the default value in the docstring but then it is hard-coded below. any reason not to put here? Same for `mask_dilations`

Reply:
Same with the other autofocus parameters, for the most part.

It's a somewhat ugly hack that resulted from me attempting to provide lots of different ways to set/override these values: in the config file (which feeds into the `Focuser ` constructor, which stores them as member variables), via the corresponding `Camera` object's `autofocus()` methods (which is what the `Observatory` calls), and via the `Focuser` object's `autofocus()` method.  The hard coded defaults only get used if there are no values set from either the config file or the call to `autofocus()`.